src/rosdiscover/interpreter/context.py

```diff
@@ -191,12 +191,6 @@ def action_server(self, ns: str, fmt: str) -> None:
         ns = self.resolve(ns)
         self._action_servers.add((ns, fmt))
 
-        self.sub(f'{ns}/goal', f'{fmt}Goal')
```

Comment:
We shouldn't get rid of these. We _may_ want to add a new tag to these `implicit` topics that are created by action servers and services. If so, we should create a new issue.

Reply:
Architecturally, I'm not sure that that is true. My thinking is that we either have an action port and none of the other ports, or the other ports and not an action port. In Acme, the architecture connector for actions has roles for clients to keep track of progress and cancel and such and so anyone that subscribes or publishes those does so architecturally through the connector.

Reply:
In any case, I've added them back in and will raise an issue once this branch is merged.

Reply:
It sounds like Acme is influencing design choices in components that are supposed to be separate. (This code and the `interpreter` stuff should be unaware of the existence of the `acme` module.) If we drop these, we'll be producing an incomplete picture of the architecture. It sounds like the issue lies not here, but in the interaction between Acme and `NodeSummary`. I strongly suspect that adding `implicit=True` will solve this issue.